ubtue_Mohr Siebeck.js

```diff
@@ -58,12 +58,14 @@ function getSearchResults(doc) {
 }
 
 function postProcess(doc, item) {
+	let subtitle = ZU.xpathText(doc, '//h2[@class="droidserif20 product-detail-h2"]');
+	if (subtitle) item.title = item.title + ': ' + subtitle;
 	if (!item.abstractNote)
 		item.abstractNote = ZU.xpathText(doc, '//div[@id="previewShort"]');
 
 	item.tags = ZU.xpath(doc, '//div[@id="productKeywords"]//a').map(i => i.textContent.trim());
 
-	item.creators = ZU.xpathText(doc, '//h2[contains(@class, "product-heading-author-block")]').split(",").map(i => ZU.cleanAuthor(i));
+	if (item.creators.length) item.creators = ZU.xpathText(doc, '//h2[contains(@class, "product-heading-author-block")]').split(",").map(i => ZU.cleanAuthor(i));
```

Comment:
In zotkat we have not had any problems so far

Reply:
Ok, but currently the logic is: overwrite the creators if they are already present but do _not_ extract them if they are empty. Maybe this works because it is always set beforehand but the original code (without `if`) seems less fragile.